client/sites-dashboard/test/utils.js

```diff
@@ -1,4 +1,5 @@
 import { isMigrationInProgress } from 'calypso/sites-dashboard/utils';
+import { getMigrationStatus, getMigrationType } from '../utils'; // Adjust the import path as necessary
```

Comment:
The `isMigrationInProgress` impart can be added here as well.

Reply:
Ah! Good catch! Fixed: https://github.com/Automattic/wp-calypso/pull/95497/commits/81f3cee6b410f4263012173f24091b06e961cf33